integration/fixtures/server-ca-csr-wildcard.json

```diff
@@ -14,6 +14,9 @@
   ],
   "CN": "example.com",
   "hosts": [
-    "*.etcd.local"
+    "*.etcd.local",
+    "etcd.local",
```

Comment:
NOTE: `"etcd.local",` can be removed with https://github.com/coreos/etcd/pull/8651. But keeping it here for older versions.